src/components/wiki/WikiPage/WikiPage.tsx

```diff
@@ -0,0 +1,47 @@
+import { MainLayout } from '@/components/MainLayout/MainLayout';
+import { MainNavigation } from '@/components/MainNavigation/MainNavigation';
+import { Heading } from '@chakra-ui/react';
+import React, { useMemo } from 'react';
+import ReadMore from '../ReadMore';
+import Tip from '../Tip';
+import { Topics } from '../Topic';
+
+interface WikiPageProps {
+	title: string;
+	content: Array<{
+		title: string;
+		description: string;
+	}>;
+	topic: Topics;
+}
+
+const WikiPage: React.FC<WikiPageProps> = ({ title, content, topic }) => {
```

Comment:
This page component is not really a good pattern to use because it will be hard to make one-off changes if needed. A better approach would be to create `WikiPageLayout`.

Then an example of a page would be something like:

```tsx

const accentColor = '#F29A33';

const DeploymentWikiPage = () => {
  return (
    <WikiPageLayout title={'Deployment'}>
      <Tip
        index={1}
        align='left'
        accentColor={accentColor}
        title='Monitor and update security controls'
        description='Regularly monitor the software for security vulnerabilities and update security controls as needed to keep the software secure.'
      />

      <Tip
        index={2}
        align='right'
        accentColor={accentColor}
        title='Keep your secrets a secret'
        description='Be careful where you choose to store your secrets and never make them publicly available. If you do, change them immediately! The DSEC tool can help you check if you have any exposed secrets in your code.'
      />

      <Tip
        index={3}
        align='left'
        accentColor={accentColor}
        title='Use secure updates'
        description='Use secure methods for updating the software, such as signed update packages and secure communications channels.'
      />

      {/* ... */}
    </WikiPageLayout>
  );
};
```

Reply:
Also notice that there is no need for `content` array which is just adding unnecessary complexity and taking away React's greatest strength - declarative programming.

With the proposed solution, very easily we can add a new Tip, change the Tip with an image, add dividers, whatever, just by adding/chaning the new component.

Reply:
Well, I agree that it is not a flexible pattern. However, this pattern allowed me to develop this as quickly as possible. As you've taken the time to provide a thorough and constructive code review, I will update it now.

Reply:
It should all be fixed now, but now I don't really see the need for even a WikiLayout, since it's essentially just a `mainLayout` with `mainNavigation`, but it is what it is.